### PR TITLE
Add Functional Tests From VMware Samples

### PR DESCRIPTION
**Pull request by MRHarrison**

### CODE REVIEW

File: docker/tests.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+cd vmware-blockchain-samples/supply-chain
+git checkout mharrison/functional-tests-on-concord
+
+echo "Building Supply Chain Sample"
+docker build -t supply-chain:latest .
+
+echo "Startup concord"
+cd ../../
+docker-compose -f docker/compose/simple4.yml up -d
+sleep 30
```
Comment (no2chem) [inline docker/tests.sh:17]: I'm a little afraid the sleep will be unreliable in a CI environment.
Can we wait for the ethRPC node to be up? (maybe use curl in a loop until we get a response?)

Comment (MRHarrison) [reply docker/tests.sh:17]: 👍 Sure no problem. 

I'm happy to see these tests pass on first try.

**Comment by no2chem** [comment]
Should we change the base branch for this PR?

**Comment by MRHarrison** [comment]
> Should we change the base branch for this PR?

Done. Looks like some artifacts were added from my rebase. I’ll fix this afternoon.

File: cmake/Findcryptopp.cmake [hunk 2]
```diff
@@ -1,21 +0,0 @@
-find_path(CRYPTOPP_INCLUDE_DIR keccak.h
```
Comment (MRHarrison) [inline cmake/Findcryptopp.cmake:1]: @no2chem  strange why this is showing up in my diff, since I see you have deleted it twice already.

Comment (no2chem) [reply cmake/Findcryptopp.cmake:1]: It's probably a macos casing issue. I think the file was renamed FindCryptopp.cmake

Comment (MRHarrison) [reply cmake/Findcryptopp.cmake]: Oh nevermind, just some artifact from rebasing off different branches, cleared it up.

**Pull request merged**